internal/passive.go

```diff
@@ -7,6 +7,8 @@ import (
 	"slices"
 	"time"
 
+	"github.com/Naman1997/discovr/verbose"
```

Comment:
This file still imports "fmt" package. Make sure only the verbose package is used, otherwise it means that we have missed some lines where we are logging stuff.